tests/lax_numpy_test.py

```diff
@@ -2662,13 +2662,8 @@ def testSortComplex(self, dtype, shape, axis):
       self.skipTest("complex sort not supported on TPU")
     rng = jtu.rand_some_equal(self.rng())
     args_maker = lambda: [rng(shape, dtype)]
-    jnp_fun = jnp.sort_complex
-    np_fun = np.sort_complex
-    if axis is not None:
-      jnp_fun = partial(jnp_fun, axis=axis)
-      np_fun = partial(np_fun, axis=axis)
-    self._CheckAgainstNumpy(jnp_fun, np_fun, args_maker)
-    self._CompileAndCheck(jnp_fun, args_maker)
+    self._CheckAgainstNumpy(jnp.sort_complex, np.sort_complex, args_maker)
```

Comment:
For now, set `check_dtypes=False` for `_CheckAgainstNumpy`.

The mismatch in your test outputs comes from an issue in the way we handle default dtypes in relation to the x64 flag... I have a WIP pull request that will fix this, but it's going to take a while to land it.

Reply:
Okay sure thing! just made the change and pushed another commit. The tests ran fine. Hopefully the CI gods will be with me!